CODEOWNERS

```diff
@@ -1,6 +1,6 @@
-qa-acct.planx-pla.net/manifest.json @themarcelor @haraprasadj @atharvar28
-qa-bloodpac.planx-pla.net/manifest.json @themarcelor @haraprasadj @atharvar28
-qa-canine.planx-pla.net/manifest.json @themarcelor @haraprasadj @atharvar28
+qa-acct.planx-pla.net/manifest.json @ac3eb @themarcelor @haraprasadj @atharvar28
```

Comment:
+1
But, before merging, please confirm with @ac3eb if he wishes to receive notifications for every change in this environment.